webapp/packages/plugin-sql-editor/src/SqlResultTabs/SqlQueryService.ts

```diff
@@ -65,6 +65,49 @@ export class SqlQueryService {
     return this.statisticsMap.get(tabId);
   }
 
+  executeEditorDatabaseDataModels(editorState: ISqlEditorTabState) {
+    const dataSource = this.sqlDataSourceService.get(editorState.editorId);
+    const databaseModels = dataSource?.databaseModels;
+
+    if (!databaseModels) {
+      console.error('databaseModels executionContext is not provided');
```

Comment:
Maybe better to throw an error as we return from function in this case

Reply:
i will remove it